Test failure: `com.linecorp.armeria.internal.common.stream.FixedStreamMessageTest`

```
Test failure: `com.linecorp.armeria.internal.common.stream.FixedStreamMessageTest.[4] stream=com.linecorp.armeria.internal.common.stream.ThreeElementFixedStreamMessage@fcdeb50`
java.lang.AssertionError: 
Expecting AtomicReference[com.linecorp.armeria.internal.common.stream.NoopSubscription@26c84006] to have value:
  com.linecorp.armeria.internal.common.stream.NoopSubscription@26c84006
but did not.
	at com.linecorp.armeria.internal.common.stream.FixedStreamMessageTest.raceBetweenSubscribeAndAbort_startSubscribeFirst_eventLoopAbortFirst(FixedStreamMessageTest.java:266)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1006)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

```

--- a/core/src/test/java/com/linecorp/armeria/internal/common/stream/FixedStreamMessageTest.java
+++ b/core/src/test/java/com/linecorp/armeria/internal/common/stream/FixedStreamMessageTest.java
@@ -263,7 +263,7 @@ class FixedStreamMessageTest {
         assertThatThrownBy(stream.whenComplete()::join)
                 .isInstanceOf(CompletionException.class)
                 .hasCause(abortCause);
-        assertThat(subscriptionRef).hasValue(NoopSubscription.get());
+        await().untilAsserted(() -> assertThat(subscriptionRef).hasValue(NoopSubscription.get()));
         assertThat(causeRef).hasValue(abortCause);
     }
 

--- a/core/src/test/java/com/linecorp/armeria/internal/common/stream/FixedStreamMessageTest.java
+++ b/core/src/test/java/com/linecorp/armeria/internal/common/stream/FixedStreamMessageTest.java
@@ -263,8 +263,10 @@ class FixedStreamMessageTest {
         assertThatThrownBy(stream.whenComplete()::join)
                 .isInstanceOf(CompletionException.class)
                 .hasCause(abortCause);
-        await().untilAsserted(() -> assertThat(subscriptionRef).hasValue(NoopSubscription.get()));
-        assertThat(causeRef).hasValue(abortCause);
+        await().untilAsserted(() -> {
+            assertThat(subscriptionRef).hasValue(NoopSubscription.get());
+            assertThat(causeRef).hasValue(abortCause);
+        });
     }
 
     @ArgumentsSource(FixedStreamMessageProvider.class)

